shared/chat/conversation/normal/container.js

```diff
@@ -5,29 +5,38 @@ import * as TrackerGen from '../../../actions/tracker-gen'
 import * as RouteTree from '../../../actions/route-tree'
 import Normal from '.'
 import {compose, connect, withStateHandlers, type TypedState} from '../../../util/container'
+import {chatTab} from '../../../constants/tabs'
 
 const mapStateToProps = (state: TypedState, {conversationIDKey}) => {
   const showLoader = !!state.chat2.loadingMap.get(`loadingThread:${conversationIDKey}`)
   const meta = Constants.getMeta(state, conversationIDKey)
-  return {conversationIDKey, showLoader, threadLoadedOffline: meta.offline}
+  const infoPanelOpen = Constants.isInfoPanelOpen(state)
+  return {conversationIDKey, infoPanelOpen, showLoader, threadLoadedOffline: meta.offline}
 }
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
   _onAttach: (conversationIDKey: Types.ConversationIDKey, paths: Array<string>) =>
     dispatch(
       RouteTree.navigateAppend([{props: {conversationIDKey, paths}, selected: 'attachmentGetTitles'}])
     ),
-  _onOpenInfoPanelMobile: (conversationIDKey: Types.ConversationIDKey) =>
-    dispatch(RouteTree.navigateAppend([{props: {conversationIDKey}, selected: 'infoPanel'}])),
+  _onToggleInfoPanel: (isOpen: boolean, conversationIDKey: Types.ConversationIDKey) => {
+    if (isOpen) {
```

Comment:
this is fine short term but ultimately might be better to have a toggle info panel action and have the saga do the right thing instead of this component know about the state and making this choice explicitly

Reply:
Yeah, I agree. That's a bit of a bigger change, I can put up something that we can let bake after the release?

Reply:
yeah lets not do this right now, just making sure people are aware of the side effects patterns and how this kind of logic likely should move